oereb_einzelschutz_denkmal/insert_oereb_einzelschutz_denkmal_tables.sql

```diff
@@ -233,8 +233,8 @@ localiseduri AS
         'de' AS alanguage,
         CASE
             WHEN rechtsvorschrften_dokument.multimedia_link IS NULL
-                THEN 'https://geo.so.ch/docs/ch.so.ada.denkmal/pdf_404.pdf'
-            ELSE regexp_replace(rechtsvorschrften_dokument.multimedia_link, 'artplus\.verw\.rootso\.org/MpWeb-apSolothurnDenkmal/download/(.*)\?mode=gis','geo.so.ch/docs/ch.so.ada.denkmal/ada_\1.pdf')
+                THEN 'https://'||${s3AdaLiveBucket}||'/pdf_404.pdf'
```

Comment:
Muss die URL nicht etwas in die folgende Richtung sein:
```suggestion
                THEN 'https://s3.eu-central-1.amazonaws.com/'||${s3AdaLiveBucket}||'/pdf_404.pdf'
```

Bitte aber verifizieren. Ich bin nicht komplett im Thema drin.

Reply:
Die URL ist korrekt so